### DISCUSSION TITLE
Collection "Translations" field conditions

### DISCUSSION THREAD

**Discussion by rynskiy** [directus/directus#15349] [Archive]
Hi

Please help me with this task:
I have:

- collection "articles"
- collection "articles_type" (connected with collection "articles" with "Many to Many" relation)
- collection "articles_translations" (automatic created by creation of field "translations")

**I need** to set condition to hide some fields by collection "articles_translations" **IF** by collection "article" field "Type" = "Post". 

With the "articles" collection, such condition work fine. But in the "articles_translations" I can't see the "Type" field of collection "articles"... 

I have tried to connect collection "articles_translations" with collection "articles" with relation "Many to Many", but still can't see the types of article by any field Conditions in "articles_translations"...

![type](https://user-images.githubusercontent.com/54798086/187891414-4887fe18-e0a1-4846-bec4-bd4e4c6da156.PNG)

**Comment by rijkvanzanten** [comment]
Conditional fields doesn't currently support nested relational data. Make sure to subscribe to https://github.com/directus/directus/issues/8228 for updates on that front 🙂
